src/toil/jobStores/googleJobStore.py

```diff
@@ -179,10 +188,28 @@ def assignID(self, jobDescription):
                   jobStoreID, '<no command>' if jobDescription.command is None else jobDescription.command)
         jobDescription.jobStoreID = jobStoreID
 
+    jobsPerBatchInsert = 200
+
+    @contextmanager
+    def batch(self):
+        self._batchedUpdates = []
+        yield
+
+        batches = [self._batchedUpdates[i:i + self.jobsPerBatchInsert] for i in
+                   range(0, len(self._batchedUpdates), self.jobsPerBatchInsert)]
```

Comment:
You should explain why you are breaking up the Toil-level batch into smaller sub-batches. If you were using a Google batch behind here, it would make sense, because I think those do have a maximum size. But as is you are just immediately looping over all these sub-batches and then each `JobDescription` in each of them, with no actual code running between the loops, so this re-shuffling  doesn't accomplish anything as is.